### ISSUE TITLE
Building tests fails with release tarball

### ISSUE THREAD

**Issue by jamessan** [msgpack/msgpack-c#685]
```
/usr/bin/c++   -I../build-gtest/install/include -I../include -Iinclude -DMSGPACK_DEFAULT_API_VERSION=3 -std=c++11 -g -O2 -fdebug-prefix-map=/<<PKGBUILDDIR>>=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2    -Wall -Wextra -MD -MT test/CMakeFiles/fuzz_unpack_pack_fuzzer_cpp11.dir/fuzz_unpack_pack_fuzzer_cpp11.cpp.o -MF test/CMakeFiles/fuzz_unpack_pack_fuzzer_cpp11.dir/fuzz_unpack_pack_fuzzer_cpp11.cpp.o.d -o test/CMakeFiles/fuzz_unpack_pack_fuzzer_cpp11.dir/fuzz_unpack_pack_fuzzer_cpp11.cpp.o -c ../test/fuzz_unpack_pack_fuzzer_cpp11.cpp
../test/fuzz_unpack_pack_fuzzer_cpp11.cpp:3:10: fatal error: ../fuzz/unpack_pack_fuzzer.cpp: No such file or directory
 #include "../fuzz/unpack_pack_fuzzer.cpp"
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

The `fuzz` directory wasn't included in the release.  I'm not sure if that was the mistake or if the fuzz test shouldn't be built by default.

**Comment by redboltz** [comment]
@jamessan , `fuzz` directory should be contained in the tar ball. I will fix it #686 and will update the release as 3.0.1.

**Comment by redboltz** [comment]
3.0.1 has been released. The problem should be fixed.
@jamessan if you still find a problem, please reopen the issue.
